images/router/f5/Dockerfile

```diff
@@ -1,11 +1,8 @@
 #
-# This is the HAProxy router for OpenShift Origin.
+# This is the F5 router for OpenShift Origin.
 #
 # The standard name for this image is openshift/origin-f5-router
 #
-FROM openshift/origin-base
+FROM openshift/origin
 
-ADD bin/openshift /usr/bin/openshift
-RUN ln -s /usr/bin/openshift /usr/bin/openshift-router
-
-ENTRYPOINT ["/usr/bin/openshift-router", "--type=f5"]
+ENTRYPOINT ["/usr/bin/openshift-f5-router"]
```

Comment:
No strong opinion either way but it just felt odd to me - to have all those symlinks stuffed in origin-base ... imho, would have gone w/ a minimalistic approach - keep the base set small and let the specializations take do/whatever they need to [it helps with the layered build approach as well].